docs/authentication.md

```diff
@@ -554,6 +594,22 @@ You can optionally import Facebook friends into Nakama's [friend graph](social-f
     }
     ```
 
+=== "Defold"
+    ```lua
+    -- Use the official Defold Facebook integration (www.defold.com/extension-facebook)
+    local permissions = { "public_profile" }
+    facebook.login_with_permissions(permissions, facebook.AUDIENCE_EVERYONE, function(self, data)
```

Comment:
could you add a comment on what the facebook audience constant does and whether a certain value is required?

Reply:
Sure, thing. I updated the code snippet and set audience to NONE (since the example only requests read permissions)